NEWS

```diff
@@ -1,15 +1,83 @@
 #
-## Copyright (C) Mellanox Technologies Ltd. 2001-2015.  ALL RIGHTS RESERVED.
-## Copyright (C) UT-Battelle, LLC. 2014-2015. ALL RIGHTS RESERVED.
-## Copyright (C) ARM Ltd. 2017-2018.  ALL RIGHTS RESERVED.
+## Copyright (C) Mellanox Technologies Ltd. 2001-2020.  ALL RIGHTS RESERVED.
+## Copyright (C) UT-Battelle, LLC. 2014-2019. ALL RIGHTS RESERVED.
+## Copyright (C) ARM Ltd. 2017-2020.  ALL RIGHTS RESERVED.
 ##
 ## See file LICENSE for terms.
 ##
 #
 
 ## Current
+## 1.8.0-rc1 (TBD)
 Features:
-- TBD
+- Improved detection for DEVX support
+- Improved TCP scalability 
+- Added ROCM perftest
+- Added optimized memcpy for ROCM devices
+- Protection from TCP connection from non-UCX applications
+- Added hardware tag-matching for CUDA buffers
```

Comment:
@yosefe Is this correct? I thought there were limitations in being able to achieve this.

Reply:
@brminich did we enable this eventually?

Reply:
we fixed a bunch of issues with that, so we can say "added/enabled")
@Akshay-Venkatesh, what limitation do you mean? 
We can post receive cuda buffers to the HW, so the data will be placed directly to it

Reply:
no, there is no such limitation